builds/uninstalling-openshift-builds.adoc

```diff
@@ -0,0 +1,31 @@
+:_content-type: ASSEMBLY
+[id="uninstalling-openshift-builds"]
+= Uninstalling {builds-shortname}
+include::_attributes/common-attributes.adoc[]
+:context: uninstalling-openshift-builds
+
+toc::[]
+
+As a cluster administrator, you can uninstall the {builds-title} Operator. Uninstalling the Operator does not automatically remove the custom resources (CRs) and {builds-shortname} components. 
+
+To uninstall, perform the following tasks:
```

Comment:
Nit/suggestion: "To uninstall the {build-shortname} Operator, perform the following tasks:"